seed/static/seed/js/services/inventory_service.js

```diff
@@ -118,6 +118,14 @@ angular.module('BE.seed.service.inventory', []).factory('inventory_service', [
      *
      */
 
+     inventory_service.properties_meter_check = function (inventory_ids) {
+       return $http.post('/api/v2/properties/meter_check/', {
```

Comment:
seems like this method would be better named meters_exist...

Reply:
Agreed. I'll jump in the PR and change that.